### ISSUE TITLE
Exception occured: SQLiteException (Can't display channels)

### ISSUE THREAD

**Issue by gnumilanix** [freshdesk/freshchat-android#9]
We are seeing the following exception log in our test Android 5.0.1 (SHV-300K) and Android 5.0.2(SM-T700) device and it's always reproducible:
```java
Exception occured
  android.database.sqlite.SQLiteException: near ",": syntax error (code 1): , while compiling: INSERT OR REPLACE  INTO categories(,_id,title,description,icon_url,position,updated_at) VALUES (,?,?,?,?,?,?);
      at android.database.sqlite.SQLiteConnection.nativePrepareStatement(Native Method)
      at android.database.sqlite.SQLiteConnection.acquirePreparedStatement(SQLiteConnection.java:1093)
      at android.database.sqlite.SQLiteConnection.prepare(SQLiteConnection.java:670)
      at android.database.sqlite.SQLiteSession.prepare(SQLiteSession.java:588)
      at android.database.sqlite.SQLiteProgram.<init>(SQLiteProgram.java:59)
      at android.database.sqlite.SQLiteStatement.<init>(SQLiteStatement.java:31)
      at android.database.sqlite.SQLiteDatabase.compileStatement(SQLiteDatabase.java:1118)
      at com.freshchat.consumer.sdk.c.i.o(Unknown Source)
      at com.freshchat.consumer.sdk.c.i.a(Unknown Source)
      at com.freshchat.consumer.sdk.service.d.e.c(Unknown Source)
      at com.freshchat.consumer.sdk.service.d.e.x(Unknown Source)
      at com.freshchat.consumer.sdk.service.c.h.a(Unknown Source)
      at com.freshchat.consumer.sdk.service.c.h.b(Unknown Source)
      at com.freshchat.consumer.sdk.service.b.b.b(Unknown Source)
      at com.freshchat.consumer.sdk.service.FreshchatService.onHandleIntent(Unknown Source)`

**Comment by tryprasannan** [comment]
@gnumilanix Can you share more information to help us understand this better like the SDK version ? And is this happening only in Samsung devices ? Is this happening on any other OS versions?

**Comment by tryprasannan** [comment]
@gnumilanix  Update: We got hold of a samsung device running Android 5.0 (SM-G900H) and weren't able to reproduce the issue from our side.

**Comment by gnumilanix** [comment]
@tryprasannan both cases were on Android 5.0.x. We are using Freshchat SDK 1.3.0. We don't really know if it's vendor specific, as we only have Android 5.0.x running on those Samsung devices.

**Comment by kashyap-virtualdusk** [comment]
getting same issue with my android tab which is running on api  22
unable to send a msg.

 E/FRESHCHAT: Exception on samsung SM-T330NU Android API 5.1.1 (22) >>>>> near ",": syntax error (code 1): , while compiling: INSERT OR REPLACE  INTO conversations(,_id,channel_id,has_pending_csat) VALUES (,?,?,?);
    #################################################################
    Error Code : 1 (SQLITE_ERROR)
    Caused By : SQL(query) error or missing database.
    	(near ",": syntax error (code 1): , while compiling: INSERT OR REPLACE  INTO conversations(,_id,channel_id,has_pending_csat) VALUES (,?,?,?);)
    #################################################################
    android.database.sqlite.SQLiteException: near ",": syntax error (code 1): , while compiling: INSERT OR REPLACE  INTO conversations(,_id,channel_id,has_pending_csat) VALUES (,?,?,?);

**Comment by tryprasannan** [comment]
@kashyap-virtualdusk  Is this your test device? or of a customer? Is it always reproducible? Would you be able to help us verify if we give you a canary version?

**Comment by kashyap-virtualdusk** [comment]
With my test device.
Yes I can. It will be really helpful.

**Comment by tryprasannan** [comment]
@kashyap-virtualdusk We've published a canary version with attempted fixes for this. Can you please test and let us know if this fixes it.

**Comment by kashyap-virtualdusk** [comment]
Awesome.
This canary version is working fine with all my test devices including tab.
Thanks a lot for this quick response.

**Comment by tryprasannan** [comment]
@kashyap-virtualdusk Thanks for the confirmation. We'll update here when we roll it out as a stable version soon.

**Comment by tryprasannan** [comment]
@kashyap-virtualdusk @gnumilanix This has been fixed and released as part of [SDK version 1.5.0](https://github.com/freshdesk/freshchat-android/releases/tag/1.5.0)
